src/nwb_conversion_tools/tools/spikeinterface/spikeinterface.py

```diff
@@ -1138,6 +1135,83 @@ def add_units(
         kwargs = {key: val[i] for key, val in aggregated_unit_properties.items()}
 
         units_table.add_unit(id=int(unit_id), spike_times=spkt, **kwargs)
+
+    if write_as == "units":
+        if nwbfile.units is None:
+            nwbfile.units = units_table
+        else:
+            warnings.warn("The nwbfile already contains units. These units will not be over-written.")
+    elif write_as == "processing":
+        ecephys_mod = get_module(
+            nwbfile=nwbfile,
+            name="ecephys",
+            description="Intermediate data from extracellular electrophysiology recordings, e.g., LFP.",
+        )
+        ecephys_mod.add(units_table)
+
+
+def add_wave_forms_to_units_table(
```

Comment:
wave_form should probably be one word, `waveform` for consistency with [the SpikeInterface WaveformExtractor](https://github.com/SpikeInterface/spikeinterface/blob/master/spikeinterface/core/waveform_extractor.py)

Reply:
Totally agree, good point!